src/tasks/lm.py

```diff
@@ -10,7 +10,7 @@
 # Fields for instance processing
 from allennlp.data import Instance, Token
 
-from ..utils.utils import process_sentence, truncate
+from ..utils.utils import process_sentence, truncate, load_tsv
```

Comment:
Why do we need this? It isn't used in the file.

Reply:
Removed, was redundant.